atoma-proxy-service/docs/openapi.yml

```diff
@@ -583,6 +583,30 @@ paths:
               schema: {}
         '500':
           description: Failed to get node distribution
+  /get_graphs:
+    get:
+      tags:
+      - Stats
+      summary: Get graphs.
+      description: |-
+        # Arguments
+
+        * `proxy_service_state` - The shared state containing the state manager
+
+        # Returns
+
+        * `Result<Json<Value>>` - A JSON response containing a list of graphs with data
```

Comment:
Just to confirm, this is all external graphs correct?

Reply:
No, that's for graphs tagged with the tag that's specified in the config.
Weirdly, when you have grafana token you can't read the shared dashboards (you can list them, but not read them).
To be more precise, imagine you have dashboard A, it has some uid uia_a, then you share it. You can still read uid_a. But you can't read shared_uid_a. Which is different. You can probably find the mapping somewhere. 
But I've decided that if everyone is on the same note, that we use tags to share. We can even have different tags for different purpose in the future. Currently we have one tag and that's it.

Reply:
That makes sense